components/hearing/Transcriptions.tsx

```diff
@@ -130,18 +174,35 @@ export const Transcriptions = ({
 
   return (
     <>
+      <SearchWrapper>
+        <SearchInput
+          type="text"
+          placeholder={
+            t("search_placeholder", { ns: "hearing" }) || "Search..."
```

Comment:
nit: It looks like we still need to update the translation json file to have these keys - placeholder/no_results are currently only available under the `search` json (for the main bill/testimony search pages), so we need to add this to the `hearing` translation json file.